src_c/event.c

```diff
@@ -1804,7 +1800,8 @@ _pg_flush_events(Uint32 type)
 static PyObject *
 pg_event_clear(PyObject *self, PyObject *args, PyObject *kwargs)
 {
-    int loop, len, type;
+    Py_ssize_t len;
```

Comment:
I know it wouldn't because that many events would never be returned... but for this loop, it would theoretically be an issue: `for (loop = 0; loop < len; loop++) {`

Because loop would wrap around into negative when it gets past sizeof(int). But there's never going to be that many events, so this isn't an issue.

Reply:
Yeah hehe. But just to know how safe we are here, I just did some quick maphs. The largest int is `2147483647` and size of `pygame.event.Event` is 72 bytes, and that turns out to be around 154 GB, so we are quite safe, no one would get 154 GB of events, the average RAM device would give up much before that